app/javascript/menu/main-menu.jsx

```diff
@@ -64,8 +67,15 @@ export const MainMenu = ({
   return (
     <>
       <div onClick={hideSecondary}>
+        <Navbar
+          isSideNavExpanded={expanded}
+          onClickSideNavExpand={() => setExpanded(!expanded)}
+          applianceName={applianceName}
+          currentUser={currentUser}
+          url={logoBrand}
```

Comment:
url sounds like an url to redirect to, maybe `logoUrl`? `brandUrl`? `logoBrand`?

Reply:
Is logoBrand a URL?  Typically when I see brand I think of an image.  I like `logoUrl` or `brandUrl`